tests/testdata/codegen/output-funcs-edgeorder/go/myedgeorder/listConfigurations.go

```diff
@@ -7,8 +7,8 @@ import (
 	"context"
 	"reflect"
 
+	"github.com/pulumi/pulumi-myedgeorder/sdk/go/myedgeorder/internal"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
-	"internal"
```

Comment:
I was wondering if this change could break someone, but just importing `internal` is broken already, so I think the answer is no.

Reply:
Aye, I think every published SDK would have been setting importBasePath anyway. This is mostly just helpful to trim our schemas for tests and to make local SDKs work.